agreements.md

```diff
@@ -63,6 +63,26 @@ Developers should also take into consideration the following rules:
 - It is recommendable for different developers to look at the subsequent iterations of the same code, as it may help catch bugs or look at the code differently.
 - If a staff member realizes that everyone has refused to review a post, they should escalate the issue and discuss it with the team.
 
+## Alerts channels
+
+The various "alerts" channels receive issue messages from running deployments. Each channel has an assignee who is responsible for monitoring messages that arrive on that channel.
+
+Similarly to the code review channel, the person responsible for a channel must properly annotate messages that indicate an issue using a proper reaction:
+- 👀 (`:eyes:`) – “I am looking at this right now”,
+- ✅ (`:white_check_mark:`) – “Issue resolved”,
+- 💤 (`:zzz:`) - “This doesn't require attention” / “This is already reported and will be addressed later”
```

Comment:
IMO superflous, I would avoid developing our own pictographic language - we already have plenty of emoji reactions in use
IMO just leaving eyes emoji or even marking as resolved (even if resolution in this case meant "created a backlog item and thats good enough").